.github/workflows/R-CMD-check.yaml

```diff
@@ -18,29 +18,48 @@ jobs:
       fail-fast: false
       matrix:
         config:
-          # - {os: macOS-latest,   r: 'release'}
-          - {os: windows-latest, r: 'release'}
-          - {os: ubuntu-latest,   r: 'devel', http-user-agent: 'release'}
-          - {os: ubuntu-latest,   r: 'release'}
-          - {os: ubuntu-latest,   r: 'oldrel-1'}
-
+          - {os: macOS-latest,   r: 'release'}
+          #- {os: windows-latest, r: 'release'}
+          - {os: ubuntu-20.04,   r: 'devel'}
+          #- {os: ubuntu-latest,   r: 'release'}
     env:
       GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
       R_KEEP_PKG_SOURCE: yes
 
     steps:
       - uses: actions/checkout@v2
+      
+      - uses: r-lib/actions/setup-r@v2
 
+      - name: Install macOS system dependencies
+        if: runner.os == 'macOS'
+        run: |
+          env HOMEBREW_NO_AUTO_UPDATE=1 brew install hdf5 proj gsl
+     
+      - uses: r-lib/actions/setup-r-dependencies@v2
+        with:
+          cache-version: 2
+          extra-packages: any::rcmdcheck
+          
       - uses: r-lib/actions/setup-pandoc@v1
```

Comment:
@adokter can this line move above `- uses: r-lib/actions/setup-r@v2` as it originally was? I think it reads easier.